app/apollo/resolvers/cluster.js

```diff
@@ -629,10 +643,15 @@ const clusterResolvers = {
               url += `&args=${arg}`;
             });
           }
+          // Allow graphQL plugins to retrieve more information
+          context.pluginContext = {uuid: updatedCluster.cluster_id, url: url, registration: updatedCluster.registration};
```

Comment:
```suggestion
          context.pluginContext = {uuid: cluster_id, name: updatedCluster.registration.name};
```
- Shouldn't need to include `url` in the pluginContext -- it's not of interest to plugins.
- should just use the `cluster_id` argument passed in
- should just include the name, not the entire `registration` object